tools/stats/import_test_stats.py

```diff
@@ -116,3 +118,14 @@ def process_disabled_test(the_response: Dict[str, Any]) -> Dict[str, Any]:
     except Exception:
         print("Couldn't download test skip set, leaving all tests enabled...")
         return {}
+
+
+def get_test_file_ratings(
+    dirpath: str, filename: str = TEST_FILE_RATINGS_FILE
+) -> Optional[Dict[str, Any]]:
+    url = "https://raw.githubusercontent.com/pytorch/test-infra/generated-stats/stats/file_test_rating.json"
```

Comment:
hmm...can we make this a bit more cross-domain repo proof?  Perhaps just move this file into a `pytorch/pytorch` folder for now.

Reply:
The two main locations for our test running jsons are test-infra and aws, I'm willing to move it to aws, but I'm not really sure why we would prefer pytorch over test-infra.  I don't think any other repo is going to try to run tests using pytorch's run_tests script?